app/src/sharedTest/java/org/oppia/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -673,4 +673,146 @@ class InputInteractionViewTestActivityTest {
     onView(withId(R.id.test_text_input_interaction_view)).check(matches(isDisplayed()))
       .check(matches(withText("abc")))
   }
+
+  @Test
+  fun testRatioInputInteractionView_withNoInputText_hasCorrectPendingAnswerType() {
+    val activityScenario = ActivityScenario.launch(
+      InputInteractionViewTestActivity::class.java
+    )
+    activityScenario.onActivity { activity ->
+      val pendingAnswer = activity.ratioExpressionInputInteractionViewModel.getPendingAnswer()
+      assertThat(pendingAnswer.answer).isInstanceOf(InteractionObject::class.java)
```

Comment:
Isn’t this always true? We should be checking that the interaction object case type is ratio expression input.

Reply:
done

Reply:
Not actually done, but I left a new comment.